modules/Bio/EnsEMBL/StopCodonReadthroughEdit.pm

```diff
@@ -0,0 +1,83 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+  Bio::EnsEMBL::StopCodonReadthroughEdit - Object representing a stop codon readthrough edit in a sequence
+
+=head1 SYNOPSIS
+
+  my $stopcodonrtedit = Bio::EnsEMBL::StopCodonReadthroughEdit->new($coord);
+
+=head1 DESCRIPTION
+
+  A representation of a stop codon readthrough edit in a sequence. 
+
+=head1 METHODS
+
+=cut
+
+package Bio::EnsEMBL::StopCodonReadthroughEdit;
+
+use strict;
+
+use parent qw(Bio::EnsEMBL::SeqEdit);
+
+=head2 new
+
+  Arg [-START/END]  : 
+       int - start and end postion of the stop codon readthrough edit in the sequence
```

Comment:
This isn't currently used, is it. That said, I *would* recommend using a named argument for passing the coordinate to the constructor - it's the way it is generally done in Ensembl and it makes the code easier to read.

As for what this argument should be named, something like POSITION or COORDINATE makes sense from the logical point of view but I would ask _e.g._ HAVANA people what term they nomally use.